classic/src/view/button/ZoomToExtent.js

```diff
@@ -0,0 +1,114 @@
+/* Copyright (c) 2015 terrestris GmbH & Co. KG
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/**
+ * ZoomToExtent Button
+ *
+ * Button used to zoom to Extent
+ *
+ * @class momo.view.button.ZoomToExtent
+ */
+Ext.define("momo.view.button.ZoomToExtent", {
+    extend: "Ext.Button",
+    xtype: 'momo-button-zoomtoextent',
+
+    requires: [
+        'momo.util.ApplicationContext',
+        'BasiGX.util.Map',
+        'Ext.app.ViewModel'
+    ],
+
+    /**
+     *
+     */
+    viewModel: {
+        data: {
+            tooltip: 'Auf Gesamtansicht zoomen',
+            text: null
+        }
+    },
+
+    /**
+     *
+     */
+    bind: {
+        tooltip: '{tooltip}',
```

Comment:
I'm not sure, if momo should get a mobile version someday. Nevertheless I will consider your suggestion here.